contracts/Auction.sol

```diff
@@ -26,12 +26,13 @@ contract Auction {
 
     bool ended; // Auction status. Default is false. Set to true at end of auction and disallows any further changes.
 
+    bid[] public bid_history;
+
     struct bid {
         address public bidder_address;
         uint public bid_amount;
         uint public time_placed;
-    }
```

Comment:
not sure if I'm reading the changes right but did you delete the closing bracket for struct bid?

Reply:
I rename the struct `bid` to `bid_record`, the naming is conflict with our `bid` function